README.md

```diff
@@ -37,19 +37,21 @@ not the actual file-system path).
 ##### dotfiles
 
 Set how "dotfiles" are treated when encountered. A dotfile is a file
-or directory that begins with a dot ("."). Note this check is done on
-the path itself without checking if the path actually exists on the
-disk. If `root` is specified, only the dotfiles above the root are
-checked (i.e. the root itself can be within a dotfile when when set
-to "deny").
+or directory that begins with a dot (".").
 
+Note: This check is only performed on the path (of the url).
+It doesn't affect `root` (of the filesystem) and doesn't actually check for dotfiles on disk.
```

Comment:
Now "It doesn't affect `root` (of the filesystem)" literally says to me "It doesn't affect / (or C:\ on Windows), because that is the root of the file system. Seeing a line like that would kind of make me worry (wait, if it doesn't affect the root of my filesystem, what would?). A callout like that would imply that there is something that does the opposite somewhere.